app/views/locations/edit_local_authority.html.erb

```diff
@@ -0,0 +1,35 @@
Harrogate", "Harrow", "Hart", "Hartlepool", "Hastings", "Havant", "Havering", "Herefordshire, County of", "Hertsmere", "High Peak", "Highland", "Hillingdon", "Hinckley and Bosworth", "Horsham", "Hounslow", "Huntingdonshire", "Hyndburn", "Ipswich", "Isle of Wight", "Isles of Scilly", "Islington", "Kensington and Chelsea", "Kettering", "King's Lynn and West Norfolk", "Kingston upon Hull, City of", "Kingston upon Thames", "Kirklees", "Knowsley", "Lambeth", "Lancaster", "Leeds", "Leicester", "Lewes", "Lewisham", "Lichfield", "Lincoln", "Liverpool", "Luton", "Maidstone", "Maldon", "Malvern Hills", "Manchester", "Mansfield", "Medway", "Melton", "Mendip", "Merton", "Mid Devon", "Mid Suffolk", "Mid Sussex", "Middlesbrough", "Milton Keynes", "Mole Valley", "New Forest", "Newark and Sherwood", "Newcastle upon Tyne", "Newcastle-under-Lyme", "Newham", "North Devon", "North East Derbyshire", "North East Lincolnshire", "North Hertfordshire", "North Kesteven", "North Lincolnshire", "North Norfolk", "North Somerset", "North Tyneside", "North Warwickshire", "North West Leicestershire", "Northampton", "Northumberland", "Norwich", "Nottingham", "Nuneaton and Bedworth", "Oadby and Wigston", "Oldham", "Oxford", "Pendle", "Peterborough", "Plymouth", "Portsmouth", "Preston", "Reading", "Redbridge", "Redcar and Cleveland", "Redditch", "Reigate and Banstead", "Ribble Valley", "Richmond upon Thames", "Richmondshire", "Rochdale", "Rochford", "Rossendale", "Rother", "Rotherham", "Rugby", "Runnymede", "Rushcliffe", "Rushmoor", "Rutland", "Ryedale", "Salford", "Sandwell", "Scarborough", "Sedgemoor", "Sefton", "Selby", "Sevenoaks", "Sheffield", "Shropshire", "Slough", "Solihull", "Somerset West and Taunton", "South Cambridgeshire", "South Derbyshire", "South Gloucestershire", "South Hams", "South Holland", "South Kesteven", "South Lakeland", "South Norfolk", "South Northamptonshire", "South Oxfordshire", "South Ribble", "South Somerset", "South Staffordshire", "South Tyneside", "Southampton", "Southend-on-Sea", "Southwark", "Spelthorne", "St Albans", "St. Helens", "Stafford", "Staffordshire Moorlands", "Stevenage", "Stockport", "Stockton-on-Tees", "Stoke-on-Trent", "Stratford-on-Avon", "Stroud", "Sunderland", "Surrey Heath", "Sutton", "Swale", "Swindon", "Tameside", "Tamworth", "Tandridge", "Teignbridge", "Telford and Wrekin", "Tendring", "Test Valley","Tewkesbury", "Thanet", "Three Rivers", "Thurrock", "Tonbridge and Malling", "Torbay", "Torridge", "Tower Hamlets", "Trafford", "Tunbridge Wells", "Uttlesford", "Vale of White Horse", "Wakefield", "Walsall", "Waltham Forest", "Wandsworth", "Warrington", "Warwick", "Watford", "Waverley", "Wealden", "Wellingborough", "Welwyn Hatfield", "West Berkshire", "West Devon", "West Lancashire", "West Lindsey", "West Oxfordshire", "West Suffolk", "Westminster", "Wigan", "Wiltshire", "Winchester", "Windsor and Maidenhead", "Wirral", "Woking", "Wokingham", "Wolverhampton", "Worcester", "Worthing", "Wychavon", "Wyre", "Wyre Forest", "York"] %>
```

Comment:
This is the same list used in the json files -  would it be worth centralising this somewhere?

Reply:
Yep, I think you could currently get it with something like `FormHandler.instance.current_form.get_question("la").answer_options.values` which would avoid duplicating the values but admittedly isn't very elegant either. I suppose if the JSON forms are refactored away it'll be easier to just store this somewhere more dedicated.

Reply:
Okay, I've used that reference, thanks - I put in some validation to ensure the user doesn't submit "Select an option" or nil as the local authority. They can still skip though, is it a requirement that they enter a local authority or can we allow them to skip and submit without one?